frontend/src/vars.scss

```diff
@@ -155,8 +155,8 @@ $z_city_background_content: 1;
 $z_city_background_image: 0;
 
 // Shadows
-$shadow_elevation: 0px 16px 16px -16px rgba(0, 0, 0, 0.25);
-$shadow_popup: 0 3px 6px -4px rgb(0 0 0 / 12%), 0 6px 16px 0 rgb(0 0 0 / 8%), 0 9px 28px 8px rgb(0 0 0 / 5%);
+$shadow_z_2: 0 4px 12px -4px rgba(0, 0, 0, 0.1), 0 16px 16px -16px rgba(0, 0, 0, 0.2);
```

Comment:
I don't know that we need multiple elevations. Generally we show 1–2 elevations at a time, and only that second elevation needs a shadow. We're also mostly limiting elevations to popovers, modals, and other overlaid content. The shadows in your example look a lot alike to me and they deviate a bit from the negative spread area in the original feedback. In cases where there are multiple layers of overlaid content, the combined shadows should be sufficient to show the stacking order. Thoughts?

Reply:
The way I see it, we have 2 elevations currently:
1. things _slightly_ above the surface, like a dashboard item card
2. things significantly above the surface that obscure things like that dashboard item card, say, menus
3. (then maybe modals, but let's not think about these now)

For me a shadow _exclusively_ at the bottom doesn't give me that feeling of elevation, as I read things top to bottom, so before I get to the bottom of the card I've already registered it as a flat thing. That's a bit of a dissonance.
Meanwhile the _menus_ obscure not just an empty backdrop like a card, but meaningful content below them, and they blend into that content much more easily with only a very light shadow. So IMO that level deserves a sligthly larger one giving that feeling of extra separation.

Reply:
It sounds like the problem you want to solve for is more separation from underlying content. I can refine the design to address that. I don't think we should go with the solution in your examples. It deviates a bit too much from the original intent, so I'll spend more time on this.

Reply:
I updated the popover examples at https://www.figma.com/file/NHPA2hVWwaKp4eTGWdigs9/?node-id=0%3A1 to have higher contrast shadows. I'm pretty set on having this shadow style for now. I think we can balance that with darker shadows to improve separation. I do not think we need to place shadows on dashboard cards or any other "flat" content. The exception would be when you're in an editing context and can drag to reorder dashboard items. I think the the border provides enough separation for now. We're going to keep changing those page styles, so I'll keep contrast and separation of content in mind as we make updates. Thoughts?

<img width="696" alt="Screen Shot 2021-11-11 at 2 37 11 PM" src="https://user-images.githubusercontent.com/254612/141378937-bd7d7d5b-5a73-4666-b2a6-ca67d4f54bf4.png">